packages/ontario-design-system-global-styles/src/styles/scss/1-variables/_tokens.variables.scss

```diff
@@ -1 +1 @@
-@forward '@ongov/ontario-design-system-design-tokens/dist/scss/variables';
```

Comment:
I believe one reason we reference the package here is to make sure that it references the package and not our file system structure when used outside out monorepo.  Do you  think this will be an issue with this update?

Reply:
Alrighty, I have reverted it so it properly forwards when used by applications